docs/integrations.md

```diff
@@ -0,0 +1,243 @@
+# Integrations
+---
+
+In order to use this feature, an additional header needs to be passed:
+
+- `Accept: application/vnd.deepcrawl.tray-io-integration-preview` 
+
+## Getting an integration by id
+
+```graphql
+{
+  node(id: "TjAxMUludGVncmF0aW9uNg") {
```

Comment:
Should we put something like this here ? 
`node(id: "<string>") {`

Reply:
the whole documentation is based on examples, I tried to keep it compliant by providing an example id. if we want to do it this way, we need to modify it everywhere. let me know.